### PR TITLE
New pricing changes - Copy of #3851

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
> COPY OF #3851 due to a misclick

## Description

This PR updates the` /pricing` page.

To directly see Dedicated pricing. you can use:

`/pricing?plan=dedicated`

## Related Issue(s)

Fixes MKWEB-94

## How to test

- Open preview
- See [`/pricing`](https://deploy-preview-3856--gitpod-kumquat.netlify.app/pricing) page :eyes:

## Release QA Checklist ✔️

<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing

-   [x] Test all links to make sure they are working correctly
-   [x] Verify that all forms (sales, support, docs feedback) are functioning properly and data is being captured correctly
-   [x] Check all buttons and calls-to-action to ensure they are leading to the correct pages
-   [x] Ensure that the website is mobile responsive and functions appropriately on different devices
-   [x] Please make sure that page is best for Light/Dark Mode. ⇒ Text/Media Changes

## [Priority: High] Content Testing

-   [x] Check all website copies for accuracy and spelling/grammar errors
-   [x] Verify that all images and videos are displaying properly
-   [x] Ensure that all downloadable files are working correctly
-   [x] Check that all external links are functioning properly

## [Priority: Medium] Performance Testing

-   [x] Try to use optimal media assets. ([Decision Record - How To](https://github.com/gitpod-io/website/blob/main/docs/adr/0003-optimal-assets-format.md))
-   [x] Test website load times to ensure that the website is loading quickly
-   [x] Verify that the website can handle high traffic volumes without crashing
-   [x] Check that all website functionality is working properly under heavy load
-   [x] Perform cross-browser testing to ensure that the website works properly on all major browsers (Google Chrome, Firefox, Edge)

## [Priority: Medium] Accessibility Testing

-   [x] Ensure that the website meets basic accessibility guidelines.
-   [x] Test that users with disabilities can easily navigate and use the website
-   [x] Verify that all website content can be accessed using assistive technologies

By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies, data storing, or analytics.

-   [x] Verify that the website is HTTPS secure and that there are no security vulnerabilities
-   [x] Ensure that all sensitive data is being stored and transmitted securely

</details>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3856"></a>

### CODE REVIEW

**Review by chrifro — APPROVED** [review]
Nice, mobile adjustments 🧡

- would it be possible to change the URL when the toggle is switched to Dedicated? Otherwise, no one will know that a specific URL exists for it. Imagine you browser through the website and what to share this specific page with a colleague
Bildschirmfoto 2023-07-14 um 08 20 03

- [ ] on dark mode: this button should be grey to not have too many primary buttons 
Bildschirmfoto 2023-07-14 um 08 22 07

File: src/lib/contents/pricing.ts [hunk 1]
```diff
@@ -13,45 +13,80 @@ import githubMarkSvelte from '$lib/components/svgs/github-mark.svelte';
...
-			'SLA and premium support',
+			'single-tenant installation for maximum security and control',
+			'data and source code isolation',
+			'access to private networking',
+			'SSO, custom SLAs and dedicated support',
 		],
-		btnText: 'Talk to sales',
-		btnHref: '/contact/sales?subject=enterprise',
+		btnText: 'Contact sales',
+		btnHref: '/contact/sales',
```
Comment (chrifro) [inline src/lib/contents/pricing.ts:57]: Any reason why you didn't use the same link as for the other contact sales button? `/contact/sales?subject=enterprise` Might not make a difference as they both link to the enterprise subject. 
Your call, I just noticed it :)

Comment (Siddhant-K-code) [reply src/lib/contents/pricing.ts:57]: yeah, it doesn't matter. Because by default it's `enterprise` only :)

**Comment by Siddhant-K-code** [comment]
Done @chrifro ✨ :)


https://github.com/gitpod-io/website/assets/55068936/0fe94c5d-2e26-41ff-9ee4-746bce4f0202

**Pull request merged**